src/plugins/type/type.c

```diff
@@ -109,14 +109,14 @@ bool elektraTypeValidateKey (Plugin * handle, Key * key, Key * errorKey)
 	const Type * type = findType (typeName);
 	if (type == NULL)
 	{
-		ELEKTRA_SET_ERRORF (ELEKTRA_ERROR_TYPE, errorKey, "Unknown type '%s' for key '%s'", typeName, keyName (key));
+		ELEKTRA_SET_VALIDATION_SEMANTIC_ERRORF (errorKey, "Unknown type '%s' for key '%s'", typeName, keyName (key));
 		return false;
 	}
 
 	if (type->normalize != NULL && !type->normalize (handle, key))
 	{
-		ELEKTRA_SET_ERRORF (ELEKTRA_ERROR_TYPE, errorKey, "The value '%s' of key %s could not be normalized (type is '%s')",
-				    keyString (key), keyName (key), typeName);
+		ELEKTRA_SET_VALIDATION_SEMANTIC_ERRORF (errorKey, "The value '%s' of key '%s' could not be normalized (type is '%s')",
```

Comment:
Isn't this a syntactic problem? (Some string is in a form that it cannot be normalized?)